app/routes/orders/new.js

```diff
@@ -29,11 +30,13 @@ export default class NewRoute extends Route {
     });
 
     const eventDetails = await order.query('event', { include: 'tax' });
-    return {
+
+    return hash({
       order,
-      event : eventDetails,
+      event      : eventDetails,
       tickets,
-      form  : await eventDetails.query('customForms', {
+      taxDetails : eventDetails.isTaxEnabled && eventDetails.get('tax', { cache: true, public: true }) || undefined,
```

Comment:
Why || undefined?

Reply:
Because , it throws 404 error if the result of getting taxDetails is not successful,  I can show that to you by removing undefined,  that's why I am returning undefined if there is no tax details.

Reply:
We can also use null in place of undefined

Reply:
thank you sir for pointing out this , i removed that